docs/data/routes/docs/nextjs/tracking-and-analytics/overview/en.md

```diff
@@ -5,11 +5,11 @@ title: Overview and architecture
 ---
 # Tracking & Analytics Overview and Architecture
 
-Full Sitecore analytics and personalization functionality is possible with Next.js on any SSR (Server-side Rendered) page route.
+Full Sitecore analytics and personalization functionality is possible with Next.js on any SSR (Server-side Rendered) page route using the [Sitecore Layout Service REST API](/docs/fundamentals/services/layout-service).
```

Comment:
Should we add the word "only" here to made it explicitly clear that tracking works *only* with SSR?